src/test/java/SignUp.java

```diff
@@ -14,16 +14,22 @@ public static void SignUpSuccess() throws InterruptedException {
         WebDriver driver = new ChromeDriver();
         driver.get("https://deens.com/?internaltraffic");
         driver.manage().window().maximize();
+        Thread.sleep(1000);
 
         driver.findElement(By.cssSelector("[href*='register']")).click();
-        driver.findElement(By.cssSelector("[id=\'username\']")).sendKeys("AlmasMendy");
         Thread.sleep(1000);
 
-        driver.findElement(By.cssSelector("[id=\'email\']")).sendKeys("someone@gmail.com");
-        Thread.sleep(1000);
+        driver.findElement(By.cssSelector("[id=\'username\']")).sendKeys("KzMendy");
```

Comment:
your test will be falls after first success passed because you need to input different username and email each time. Or you need to clean database after each run